setup.py

```diff
@@ -58,7 +58,8 @@ def get_all_extras() -> Dict:
     "pyyaml>=4.2b1,<6.0",
     "requests>=2.22.0,<3.0.0",
     "python-dotenv>=0.14.0,<0.18.0",
-    "ecdsa>=0.15,<0.17.0"
+    "ecdsa>=0.15,<0.17.0",
+    "asn1crypto>=1.4.0,<=1.5.0",
```

Comment:
Why not `>=1.4.0, <=2.0.0`?

Reply:
not sure about how this library is going to be updated, i think this version restriction is ok for the moment